docs/release-notes/8.7.asciidoc

```diff
@@ -5,6 +5,94 @@
 [[release-notes-8.7.1]]
 === 8.7.1
 
+[discrete]
+[[known-issue-8.7.1]]
+==== Known issues
+
+* If you modify an exception item using the <<exceptions-api-update-item,update exception item>> API and _only_ specify its `item_id`, the exception item is erroneously duplicated. To avoid this issue, you can:
+
+** <<manage-exception,Update exception items>> through the {security-app} UI. 
+** Specify an exception item's `item_id` _and_ its `id` when modifying an exception through the <<exceptions-api-update-item,update exception item>> API. 
+** Only specify an exception item's `id` when updating an exception through the update exception item API. 
```

Comment:
I don't think that we want to recommend this one; I describe in [https://github.com/elastic/kibana/issues/159230] the caveats with that approach, which is that the `item_id` gets lost (so, a very similar bug).

Reply:
Ah, was the bug described in https://github.com/elastic/kibana/issues/159230 not fixed in https://github.com/elastic/kibana/pull/159223?

Reply:
Yep! It fixes both issues.